lib/cell.h

```diff
@@ -16,8 +16,10 @@ namespace dsl {
  * The cell can also contain a generic data of type T, which by default is just
  * a point in the cell.
  */
-template < class PointType, class DataType >
+template < class PT, class DT >
```

Comment:
I do it because I typedef them again as PointType and DataType so that it is accessible for use elsewhere like so
`using CarTwistConnectivity     = SE2GridConnectivity<SE2Cell::PointType, SE2Cell::DataType, SE2Twist>;`
`using CarPathConnectivity      = SE2GridConnectivity<SE2Cell::PointType, SE2Cell::DataType, SE2Path>;`

What do you suggest I do?